core/src/cgcloud/core/cluster.py

```diff
@@ -138,7 +138,8 @@ def __init__( self, ctx ):
 
     def _set_instance_options( self, options ):
         super( ClusterWorker, self )._set_instance_options( options )
-        self.leader_instance_id = options.get( 'leader_instance_id' )
+        self.leader_instance_id = options.get( 'leader_instance_id', '' )
```

Comment:
Can you elaborate on your intentions a little bit? It's been a while since I worked on that code.

Reply:
Why is the empty string a better default than `None`?